src/content/fr/2020/performance.md

```diff
@@ -0,0 +1,388 @@
+---
+part_number: II
+chapter_number: 9
+title: Performance
+description: Chapitre Performance du Web Almanac 2020 couvrant les Signaux Web Essentiels (Core Web Vitals), le score de performance de Lighthouse, le First Contentful Paint (FCP) et le Time to First Byte (TTFB).
+authors: [thefoxis]
+reviewers: [borisschapira, rviscomi, obto, noamr, Zizzamia]
+analysts: [max-ostapenko, dooman87]
+editors: [bazzadp]
+translators: [borisschapira]
+thefoxis_bio: Karolina est <em lang="en">Product Design Lead</em> chez <a hreflang="en" href="https://calibreapp.com/" lang="en">Calibre</a>, travaillant à la création de la plate-forme de surveillance de la vitesse la plus complète. Elle réalise la curation de la <a hreflang="en" href="https://perf.email/" lang="en">Performance Newsletter</a>, votre source d’informations et de ressources sur la performance. Karolina <a hreflang="en" href="https://calibreapp.com/blog/category/web-platform">écrit aussi fréquemment</a> sur la façon dont la performance affecte l’expérience d’utilisation.
+discuss: 2045
+results: https://docs.google.com/spreadsheets/d/164FVuCQ7gPhTWUXJl1av5_hBxjncNi0TK8RnNseNPJQ/
+queries: 09_Performance
+featured_quote: De mauvaises performances ne se traduisent pas seulement par de la frustration ou des effets négatifs sur les objectifs commerciaux, elles créent des véritables barrières à l’entrée. La pandémie mondiale de cette année a rendu ces obstacles encore plus évidents.
+featured_stat_1: 25&nbsp;%
+featured_stat_label_1: de sites avec un bon FCP sur mobile
+featured_stat_2: 18&nbsp;%
+featured_stat_label_2: de sites avec un bon TTFB sur mobile
+featured_stat_3: 4&nbsp;%
```

Comment:
Since you're not using `&nbsp;` in other places should we just use the unecoded non-breaking space here too for consistency?:

```suggestion
featured_stat_1: 25 %
featured_stat_label_1: de sites avec un bon FCP sur mobile
featured_stat_2: 18 %
featured_stat_label_2: de sites avec un bon TTFB sur mobile
featured_stat_3: 4 %
```

Sorry if that's what you had before - I didn't realise it!